client/src/Account/components/AccountPreviousRewards.tsx

```diff
@@ -177,12 +185,14 @@ const AccountPreviousRewards: FC<AccountPreviousRewardsProps> = ({ rewards }) =>
           return parseFloat(previousRewards.geminiIII.IIIf.earnings).toFixed(2)
         case 'gemini3g':
           return formatUnitsToNumber(
-            rewards
-              .reduce(
-                (acc, reward) => (reward.amount ? acc + BigInt(reward.amount) : acc),
-                BigInt(0),
-              )
-              .toString(),
+            rewards && rewards.length > 0
+              ? rewards
+                  .reduce(
+                    (acc, reward) => (reward.amount ? acc + BigInt(reward.amount) : acc),
+                    BigInt(0),
+                  )
+                  .toString()
+              : '0',
```

Comment:
There are accounts with > 100,000 rewards, I don't think calculating the total on the fly will be a good solution. Also, I believe this query will also return `OperatorRewards`, which should not be included. The best short-term solution may be to have Gemini 3g as "coming soon" until we have the reward totals calculated. We will also probably want to include any rewards from Stake Wars as a separate line item.

Reply:
Also in the API we have an aggregator entity that you can use the get the rewards by account: 

example:
```
query MyQuery {
  accountRewards(where: {account: {id_eq: "st7MzhqFYzdrC1hQ5UUp2UK4xNfCM9P9E66mZdYJPTdRvRqEY"}}, limit: 10) {
    amount
    block
    id
    operator
    updatedAt
    vote
  }
}
```

Reply:
@natachadelarosa does `operator` also include nominator rewards?

Reply:
You can think of `operator` as nominator rewards since the operator rewards are distributed among the nominators with the reduction on the `operator-tax` and added to this operator aggregator by account

Reply:
@jfrank-summit So do we prefer to simply leaving 3g as `coming soon` or using the example of @natachadelarosa I'm understanding that stake war reward is `operator` so I could make a separate phase called “Stake Wars” and for the regular 3g phase `amount - operator`

Reply:
The aggregator won't be accurate once we finish incentivization for block+vote rewards, but as long as we are on top of that I am ok using it. It is already not accurate for operator rewards, as it will include rewards outside of stake wars. Also, stake wars calculations are a bit trickier, @natachadelarosa will be working on a separate indexer for them.